### ISSUE TITLE
Speech recognition Template on New Dataset

### ISSUE THREAD

**Issue by monaabdelazim** [speechbrain/speechbrain#1141]
I am trying to run the ASR template on my own data. I have generated the tokenizer and the language model as mentioned in the tutorial . when I start training the model the RIRS dataset starts downloading. How to convience the toolkit to start training on my own dataset and to generate the LM and tokenizer ckpt files?
The log files are attached.
[log.txt](https://github.com/speechbrain/speechbrain/files/7566393/log.txt)
[env.log](https://github.com/speechbrain/speechbrain/files/7566398/env.log)

**Comment by mravanelli** [comment]
Hi @monaabdelazim,
if you would like to perform train, validation, and test of an ASR model with your own data you have to create your annotation files (in JSON format). Basically, you have to create train.json, valid.json, test.json  such that they contain the audio files that you want to use (https://github.com/mravanelli/speechbrain-1/blob/develop/templates/speech_recognition/ASR/train.yaml#L43). You can take a look at the json files created when running the template to figure out about their expected structure.

**Comment by monaabdelazim** [comment]
Hiv@mravanelli,
I have already created the mentioned files for my dataset. But when I start running the training using the mentioned command python train.py train.yaml
It starts downloading the RIrs dataset.
I guess I need to modify a certain parameter in the .yaml file or adding a certain attribute to the command but I am totally confused.can you help me with that issue?

**Comment by mravanelli** [comment]
Hi,
just comment the environmental corruption part (
https://github.com/mravanelli/speechbrain-1/blob/develop/templates/speech_recognition/ASR/train.yaml#L131)
in the yaml if you don't want environmental corruption with your own
dataset.

**Comment by monaabdelazim** [comment]
@mravanelli I did but I recieved another error.. value error: the reference "env_corrupt" is not valid. How to get rid of this error?

**Comment by mravanelli** [comment]
Right, you have to comment also the corresponding part in train.py (lines
142-145
https://github.com/speechbrain/speechbrain/blob/develop/templates/speech_recognition/ASR/train.py#L142
)

**Comment by monaabdelazim** [comment]
What about lines 167-169? Shall I comment them too?

**Comment by mravanelli** [comment]
Yes, all the entries with environmental augmentation (if, for some reason,
you don't want it)

**Comment by monaabdelazim** [comment]
@mravanelli I followed your instructions and this is my new question. How to provide the training command with the LM and tokenizer created using my dataset(i.e is there such a parameter to add to the training command?)?
